File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/For/Exercício 04.py
#Exercício - Dentro Fora

dentro = 0
fora = 0

n = int(input("Quantos números você vai digitar? "))

for i in range(n):
    
    x = int(input("Digite um número: "))

    if x >= 10 and x <= 20:
        dentro = dentro + 1
    else:
        fora = fora + 1

print(f"{dentro} dentro")
print(f"{fora} fora")

File: 07 - C++/Exercícios/03 - Estruturas Repetitivas/For/Exercício 04.cpp
//Exercício - Dentro Fora

#include <iostream>

using namespace std;

int main(){

    int i, N, x, dentro, fora;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    dentro = 0;
    fora = 0;

    for(i = 1; i <= N; i++){

        cout << "Digite um numero: ";
        cin >> x;

        if(x < 10 || x > 20){
            fora = fora + 1;
        }
        else{
            dentro = dentro + 1;
        }
    }

    cout << dentro << " dentro" << endl;
    cout << fora << " fora" << endl;

    return 0;

}